airbase/pom.xml

```diff
@@ -176,7 +176,7 @@
         <dep.packaging.version>0.163</dep.packaging.version>
 
         <!-- Dependency versions that should be the same everywhere. -->
-        <dep.guice.version>4.2.2</dep.guice.version>
+        <dep.guice.version>5.0.1</dep.guice.version>
```

Comment:
Can this be upgraded to [6.0.0](https://github.com/google/guice/wiki/Guice600). Since we use `javax.Inject` this may be the highest version we can go

Reply:
+1 to @aaneja  
hi @imjalpreet is the build ok with 6.0.0?  thanks!